src/components/Transaction/AddTransaction.jsx

```diff
@@ -0,0 +1,199 @@
+import React, { Component } from 'react';
```

Comment:
Should we add transactions directly, or should we be adding this to the product page?

Reply:
I am not sure. What do you think?

Reply:
I think it's probably better to add a transaction log whenever something is done on the product page

Reply:
So we don't need add transaction in transaction page? I can remove it.